flux_local/git_repo.py

```diff
@@ -412,12 +428,17 @@ def __init__(self) -> None:
         self._cache: dict[str, kustomize.Kustomize] = {}
 
     async def build(
-        self, kustomization: Kustomization, path: Path
+        self,
+        kustomization: Kustomization,
+        path: Path,
+        ignore_paths: list[str],
     ) -> kustomize.Kustomize:
-        key = f"{kustomization.namespaced_name} @ {path}"
+        key = f"{kustomization.namespaced_name} @ {path} ! {','.join(ignore_paths)}"
```

Comment:
My impression is that ignore paths is always passed with the same value so it doesn't really need to be part of the cache key? or do we pass in different values? Can it be passed to the constructor?